docs/rest/FhirPatchRequests.http

```diff
@@ -337,4 +337,65 @@ Authorization: Bearer {{bearer.response.body.access_token}}
             ]
         }
     ]
+}
+
+### Add a security tag. This will always add the elements to the end of the array if the array exists or not.
+PATCH https://{{hostname}}/Patient/{{patient.response.body.id}}
+content-type: application/fhir+json
+Authorization: Bearer {{bearer.response.body.access_token}}
+
+{
+  "resourceType": "Parameters",
+  "parameter": [
+    {
+      "name": "operation",
+      "part": [
+        {
+          "name": "type",
+          "valueCode": "add"
+        },
+        {
+          "name": "path",
+          "valueString": "Patient.meta"
+        },
+        {
+          "name": "name",
+          "valueString": "security"
+        },
+        {
+          "name": "value",
+          "valueCoding": {
+            "system": "http://example.org/security-system",
+            "code": "SECURITY_TAG_CODE",
+            "display": "New Security Tag Display"
+          }
+        }
+      ]
+    },
+    {
+      "name": "operation",
```

Comment:
What does having 2 values add to the example?

Reply:
It shows that one operation doesn't replace the array if it already exists. Would it make sense in two requests?

Reply:
Ah I see. That makes sense.